features/networking/sgw-lgw-migration.feature

```diff
@@ -186,7 +186,7 @@ Feature: SGW<->LGW migration related scenarios
   @4.11 @4.10
   @admin
   @destructive
-  @network-ovnkubernetes
+  @network-ovnkubernetes @network-openshiftsdn
```

Comment:
could you remove `@network-openshiftsdn` tag,  I know it's not your tools issue. it's case parameter have `ALL`. I just update this in Polartion

Reply:
@zhaozhanqi Updated.

Reply:
Thanks